packages/i18next/src/I18nextNavigationLabel.tsx

```diff
@@ -0,0 +1,20 @@
+import type { i18n } from "i18next";
+import { useTranslation } from "react-i18next";
+
+export interface I18nextNavigationLabelProps {
+    i18next: i18n;
+    namespace?: string;
+    resourceKey: string;
+}
+
+export function I18nextNavigationLabel(props: I18nextNavigationLabelProps) {
+    const {
+        i18next,
+        namespace = "navigationItems",
```

Comment:
That seems like a big assumption that the namespace would be "navigationItems" here. I would probably not put a default here

Reply:
In react-i18next the default namespace is "common" if I remember correctly, I would've expected the same here.

Reply:
Make sense! In this case thought this is a default namespace for a specific use case that only apply to navigation items so IMO, having `navigationItems` as a default namespace is still a valid choice.